public/javascripts/playSettings.js

```diff
@@ -39,10 +52,15 @@ function pixelateImage(originalImage, pixelationFactor) {
 }
 
 function chooseDifficulty(radio) {
-  chosenDifficulty = parseInt(radio.value);
-  const minImageDimmension = img.width < img.height ? img.width : img.height;
-  const pixelSize =
-    Math.floor(minImageDimmension / (60 / chosenDifficulty)) + 1;
-  const newImg = pixelateImage(imgCopy, pixelSize);
-  document.getElementById("pixelatedImage").src = newImg;
+  const chosenDifficulty = parseInt(radio.value);
+  if (!sessionDifficultyArr.includes(chosenDifficulty)) {
+    console.log("This is not a valid value");
+    radio.checked = false;
+  } else {
+    const minImageDimmension = img.width < img.height ? img.width : img.height;
+    const pixelSize =
+      Math.floor(minImageDimmension / (60 / chosenDifficulty)) + 1;
```

Comment:
In the future, it would be nice to leave some comments in cases like this explaining how the calculation is 
done, otherwise it's just random numbers for others.